setup.cfg

```diff
@@ -1,23 +1,3 @@
-[flake8]
-max-line-length = 120
-exclude = docs/src, build, dist, .ipynb_checkpoints
-ignore = W503,E203
-per-file-ignores =
-    numpyro/contrib/tfp/distributions.py:F811
-    numpyro/distributions/kl.py:F811
-
-[isort]
-profile = black
-skip_glob = .ipynb_checkpoints
-known_first_party = funsor, numpyro, test
-known_third_party = opt_einsum
-known_jax = flax, haiku, jax, optax, tensorflow_probability
-sections = FUTURE, STDLIB, THIRDPARTY, JAX, FIRSTPARTY, LOCALFOLDER
-force_sort_within_sections = true
-combine_as_imports = true
-multi_line_output = 3
-skip=docs
-
 [tool:pytest]
```

Comment:
Do you think that we can also move pytest config to the pyproject file. I saw it is supported [here](https://docs.pytest.org/en/7.3.x/reference/customize.html#pyproject-toml). (not necessary in this PR though)

Reply:
Indeed! I want to do this in another PR. In addition I wanna add pre-commit hooks :) (also in a different PR)